---
title: "Schedule"
date: 2024-10-02T10:48:14+01:00
---

Day 1 (26th November)
---------------------

|  | Time | Speaker | Topic |
|---|---|---|---|
|  |  |  |  |
| Conference opening | 1:30pm | John Lees |  |
|  |  |  |  |
| Opening talk | 1:45pm | Christophe Fraser | Transmission studies and epidemiology of HIV informed from analyses of within and between host pathogen diversity |
|  |  |  |  |
| **Session 1: Metagenomics** | 2:15pm | Sarah Buddle | Evaluating metagenomics and targeted approaches   for diagnosis and surveillance of viruses       |
|  | 2:35pm | Guillaume Meric | Exploring links between microbial infections, gut microbes and health   using large datasets |
| Break | 2:55pm |  |  |
|  | 3:10pm | Tommi Maeklin | Linking colibactin-producing E. coli prevalence in metagenomics data with   colorectal cancer incidence |
|  | 3:20pm | Sana Naderi | Calculating population genetic metrics from wastewater sequencing data to   predict SARS-CoV-2 waves |
|  | 3:30pm | Tom Stanton | Enabling antigen structure prediction with Kaptive for enhanced vaccine   development |
|  | 3:40pm | Gerry Tonkin-Hill | Tracking strain persistence and transmission using metagenomics |
|  |  |  |  |
| **Lightning talks I** | 4:10pm |  |  |

Day 2 (27th November)
---------------------

|  | Time | Speaker | Topic |
|---|---|---|---|
| **Session 2: Selection and evolution** | 10:30am | Liam Shaw | The bacterial pan-immune system and selective pressures on plasmids |
|  | 10:50am | Rebecca Gladstone | Diversification of ABC transporter-dependent capsular K-loci in E. coli  |
|  | 11:10am | Gherard Batisti Biffignandi | Genes on the lam: how fast can AMR-MGEs spread in a bacterial population? |
| Break | 11:20am |  |  |
|  | 11:50am | Adam Dinan | A pan-transcriptomic view of bacterial evolution |
|  | 12:00am | Sam Horsfield | A Large Language Model for studying bacterial genome architecture,   population structure and epidemiology |
|  | 12:10am | Evangelos Mourkas | Evolution of high resistance in Campylobacter in Peru |
|  |  |  |  |
| Lunch | 12:30pm |  |  |
|  |  |  |  |
| **Session 3: Emerging infectious diseases and pandemics** | 3:30pm | Jumpei Ito | Predicting Viral Antigenicity, Fitness, and Evolution |
|  | 3:50pm | Gemma Murray | The emergence of a zoonotic bacterial pathogen from within the pig   respiratory microbiota |
|  | 4:00pm | Rebecca French |  Comprehensive infectome analysis   of an emerging disease in the critically endangered Kakapo using total RNA   sequencing |
|  | 4:10pm | Eunice Njuguna | The Impact of Pneumococcal Conjugate Vaccine (PCV10) on Pneumococcal   Population Structure in Kilifi, Kenya: A Whole Genome Sequencing Analysis |
| Break | 4:20pm |  |  |
|  | 4:40pm | Harry Chown | Utilising pangenomics to study the human fungal pathogen Aspergillus   fumigatus |
|  | 4:50pm | Jemma Geoghegan | Tracking SARS-CoV-2 in New Zealand: insights from genomics |
| **Lightning talks II** | 5:10pm |  |  |
|  |  |  |  |
| **Standing dinner** | 6:00pm |  |  |

Day 3 (28th November)
---------------------
|  | Time | Speaker | Topic |
|---|---|---|---|
| **Session 4: Pangenomes and tools** | 10:30am | Leah Roberts | Using pangenomes to investigate plasmid evolution on a miniscule   timescale |
|  | 10:50am | Chao Yang | Macrogenetic Atlas of Prokaryotes |
|  | 11:00am | Jane Hawkey | Developing a robust approach to determining plasmid transmission in the   hospital setting |
| Break | 11:10am |  |  |
|  | 11:30am | Caitlin Collins | Molecular clocks in the accessory genome?: Investigating rates of gene   gain, loss, and selection |
|  | 11:40am | Daria Frolova | Applying rearrangement distances to enable plasmid epidemiology with   pling |
|  | 11:50am | Marco Molari | Evolution of genome structure and content in closely-related bacteria |
|  |  |  |  |
| Lunch | 12:10pm |  |  |
|  |  |  |  |
| **Session 5: Modelling** | 2pm | Hsiao-Han Chang | Using Identical by Descent (IBD) to Infer Infectious Disease   Transmission: What Does IBD Really Mean? |
|  | 2:20pm | Danielle Ingle | Bacterial phylodynamics of drug-resistant pathogens |
|  | 2:40pm | Jonathan Pekar | The recency and geographical origins of the bat viruses ancestral to   SARS-CoV and SARS-CoV-2 |
| Break | 2:50pm |  |  |
|  | 3:10pm | Chris Ruis | Inferring host species of pathogens through mutational spectra |
|  | 3:20pm | Seungwon Ko | Epistasis analysis across bacterial pangenomes using Continuous-Time   Markov Models |
|  | 3:30pm | Nicholas Croucher | How intragenomic conflict shapes the pangenome |
|  |  |  |  |
| Close | 4pm |  |  |
